### PR TITLE
DIV-4523 Add webchat to landing pages and footers in PFE, RFE, DN, DA

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-4523 -Add webchat to landing pages and footers in PFE, RFE, DN, DA ](https://tools.hmcts.net/jira/browse/DIV-4523)
 - Add webchat behind feature toggle
 - Update content regarding contact information at bottom of the page

### CODE REVIEW

File: charts/div-dn/values.yaml [hunk 1]
```diff
@@ -11,6 +11,11 @@ nodejs:
         RATE_LIMITER_ENABLED: "false"
         RATE_LIMITER_EXPIRE: "3600000"
         RATE_LIMITER_TOTAL: "3600"
+        WEBCHAT_CHAT_ID: 3833071605d5d4518036a09.30917386
```
Comment (yarpo) [inline charts/div-dn/values.yaml]: I assume these values were given by webchat team?

Comment (tchow8) [reply charts/div-dn/values.yaml]: Also, are these fine to be public or should they be secrets?

Comment (dtaylor7) [reply charts/div-dn/values.yaml]: > I assume these values were given by webchat team?
yes they were

> Also, are these fine to be public or should they be secrets?
these are publicly available so no need to put in vault

**Comment by dtaylor7** [comment]
> Is this whitelisted by Webchat team already? They have contacted me to get urls where webchat will be working on.

Yes i have provided all relevant urls to Eoghan

**Comment by vianvi** [comment]
need to close PR and reopen it again because of idam issue

**Pull request closed without merging**